tutor/src/components/tabs.cjsx

```diff
@@ -75,15 +75,12 @@ Tabs = React.createClass
   renderTab: (tab, index) ->
     isSelected = index is @state.activeIndex
 
-    <li key={index} tabIndex={index} className={classnames(active: isSelected)}>
-      <a role="tab"
-        href="#"
-          tabIndex={if isSelected then -1 else 0}
-        onClick={_.partial(@onTabClick, index)}
+    <li key={index} tabIndex={if isSelected then -1 else 0} className={classnames(active: isSelected)} role="tab">
+      <h2 onClick={_.partial(@onTabClick, index)}
         aria-selected={'true' if isSelected}
       >
         {tab}
-      </a>
+      </h2>
     </li>
 
   render: ->
```

Comment:
The `Tabs` component is used on several other areas, `course-settings/roster`,  and `course-periods` which is in turned used from `cc-dashboard`, `performance-forecast` and `plan-stats`

Is UX ok with all of them getting much larger?